core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -64,7 +64,7 @@ object Runtime {
       .getOrElse(JupyterService)
 
     new URL(
-      urlBase + googleProject.value + "/" + runtimeName.asString + "/" + tool.proxySegment
+      urlBase + cloudContext.asString + "/" + runtimeName.asString + "/" + tool.proxySegment
```

Comment:
I think this only works for the Google case, right? Just wondering if it should fail if it's an Azure cloud context for now.

Reply:
will do